docs/typescript-support.md

```diff
@@ -0,0 +1,67 @@
+# Typescript Support
+
+Styled-components has typescript definitions to allow the library to be
```

Comment:
To adhere to the style of the docs generally, could you make all mentions of `styled-components` in text look like code by doing this:

```
`styled-components`
```